source/core/ut_suite_builder.pkb

```diff
@@ -560,8 +567,13 @@ create or replace package body ut_suite_builder is
     if a_annotations.by_name.exists(gc_tags) then
       add_tags_to_suite_item(a_suite, a_annotations.by_name(gc_tags),a_suite.tags);
     end if;
+    
     a_suite.disabled_flag := ut_utils.boolean_to_int(a_annotations.by_name.exists(gc_disabled));
-
+    if a_annotations.by_name.exists(gc_disabled) then
+      l_annotation_text := trim(a_annotations.by_name(gc_disabled)(a_annotations.by_name(gc_disabled).first));
```

Comment:
Is trimming needed? Value is trimmed here, but not trimmed in line 419.
Can we make this consistent? 
No need to use `l_annotation_text` variable. Value can be assigned directly to `a_sute.disabled_reason`